pkgs/applications/science/logic/formula/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, buildDotnetModule, dotnet-sdk_5 }:
+
+buildDotnetModule rec {
+  pname = "formula-dotnet";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner = "VUISIS";
+    repo = "formula-dotnet";
+    rev = "60dd3d91d931a795fa9fe6b6054f63e03cec9d28";
+    sha256 = "sha256-7odYV4FtImzX3vMUfO7sPlXkw6mOkZOFbnQLpJDXJc4=";
+  };
+
+  nugetDeps = ./nuget.nix;
+  projectFile = "Src/CommandLine/CommandLine.csproj";
+
+  makeWrapperArgs = "--set PATH ${dotnet-sdk_5}/bin";
+
+  installPhase = ''
```

This overrides the installation `buildDotnetModule` is meant to do, making the only thing that gets installed be the wrapper. You could do this in `preFixup` instead, which should fix the issue:
```suggestion
  preFixup = ''
```

Seeing how the issue doesn't occur on all platforms, I think it should be conditional as well.